src/validation.cpp

```diff
@@ -3828,6 +3828,7 @@ bool ChainstateManager::AcceptBlockHeader(const CBlockHeader& block, BlockValida
         return state.Invalid(BlockValidationResult::BLOCK_HEADER_LOW_WORK, "too-little-chainwork");
     }
     CBlockIndex* pindex{m_blockman.AddToBlockIndex(block, m_best_header)};
+    LogPrint(BCLog::VALIDATION, "%s: added new block header %s\n", __func__, hash.ToString());
```

Comment:
```suggestion
    LogPrint(BCLog::VALIDATION, "added new block header %s\n", hash.ToString());
```

We have an option for logging source locations `-logsourcelocations`.

Reply:
Done!